### PR TITLE
Memfix

### PR DESCRIPTION
**Pull request by darafferty**

### CODE REVIEW

File: rapthor/pipeline/parsets/image_pipeline.cwl [hunk 1]
```diff
@@ -245,9 +245,9 @@ inputs:
     type: float[]
 
   - id: wsclean_mem
-    label: Memory fraction
+    label: Memory percentage
     doc: |
-      The memory fraction limit for WSClean (length = n_sectors).
+      The memory limit for WSClean in percent of total (length = n_sectors).
```
Comment (gmloose) [inline rapthor/pipeline/parsets/image_pipeline.cwl:250]: Do you still want to keep this a float, now that you changed fraction to percentage?

File: rapthor/pipeline/parsets/image_sector_pipeline.cwl [hunk 2]
```diff
@@ -220,9 +220,9 @@ inputs:
     type: float
 
   - id: wsclean_mem
-    label: Memory fraction
+    label: Memory percentage
     doc: |
-      The memory fraction limit for WSClean (length = 1).
+      The memory limit for WSClean in percent of total (length = 1).
```
Comment (gmloose) [inline rapthor/pipeline/parsets/image_sector_pipeline.cwl:225]: See my previous remark in image_pipeline.cwl

File: rapthor/pipeline/steps/wsclean_image.cwl [hunk 3]
```diff
@@ -146,9 +146,9 @@ inputs:
     inputBinding:
       prefix: -taper-gaussian
   - id: wsclean_mem
-    label: Memory fraction
+    label: Memory percentage
     doc: |
-      The memory fraction limit.
+      The memory limit in percent of total.
```
Comment (gmloose) [inline rapthor/pipeline/steps/wsclean_image.cwl:151]: See my previous comment in image_pipeline.cwl

**Review by gmloose — APPROVED** [review]

File: rapthor/pipeline/parsets/image_pipeline.cwl (hunk 1 above)
Comment (darafferty) [reply rapthor/pipeline/parsets/image_pipeline.cwl:250]: Good point -- an int would work fine too, but I don't think it really matters in practice. By "length = n_sectors", I just mean the length of the input list should be n_sectors long, where n_sectors is the number of sectors; however, it's not really necessary to have a list here for the memory limit (as you probably don't need a different limit for each sector), but it does no harm either I guess.

**Pull request merged**